lib/clAdapters/activeSync/calendar/index.ts

```diff
@@ -0,0 +1,310 @@
+import * as moment from 'moment';
+import * as _ from 'lodash';
+import { Configuration, Service } from '../../base/index';
+import * as asclient from 'asclient';
+import ActiveSyncBaseAdapter from '../base/Adapter';
+import autodiscover from 'autodiscover-activesync';
+
+const credentialMappings: { [key: string]: string } = {
+  'username' : 'username',
+  'email'    : 'email',
+  'password' : 'password',
+};
+
+const ACCEPTED_STATUS: string = '3';
+
+export const fieldNameMap = {
+  // Desired...                          // Given...
+  // ? :                                 'TimeZone', // Do we need this?  I don't think so...
+  'eventId':                             'UID',
+  'attendees':                           'Attendees',
+  'dateTimeCreated':                     'DtStamp',
+  'attendeeAddress':                     'Email',
+  'attendeeName':                        'Name',
+  // 'iCalUId':                             'iCalUID', // Does not appear to be available
```

Comment:
We do need the iCalUId in addition to eventId for identifying recurring event series. Don't think this will work with Daily Activities for recurring events without this and seriesMasterId. :(